examples/discovery-search-app/cypress/integration/dynamic_facets/spec.ts

```diff
@@ -83,6 +83,6 @@ describe('Dynamic Facets', () => {
     // makes a query with both filters', () => {
     cy.wait('@postQueryDynamicAndNonDynamic')
       .its('requestBody.filter')
-      .should('eq', 'location:"Hancock, MN","regression"');
+      .should('eq', 'location:(Hancock\\, MN),(regression)');
```

Comment:
What's up with the double-backslash?

Reply:
Double backslash is interpreted in a string as the backslash character. So we'll send `\,` in the query, so that the comma won't get parsed as part of the query (since commas are query operators).

Reply:
I still need some clarity about exactly which characters need escaping. For the moment, I've set us up to escape all [query operator](https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-query-operators) characters.